### ISSUE TITLE
Un-blocking media still shows missing.png for avatar image

### ISSUE THREAD

**Issue by nolanlawson** [tootsuite/mastodon#5781]
### Steps to reproduce:

1. Block a domain with severity "None" and "Reject media" checked
2. Un-block the domain
3. Look at the profile of a now-unblocked user

### Expected

It shows their avatar image.

### Actual

It shows `missing.png`.

I tried restarting the server and it's still showing `missing.png`... :confused:

Mastodon: v2.0.0

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by nolanlawson** [comment]
Re-silencing and then checking "Unsilence all existing accounts from this domain" when undoing also don't seem to fix it.

**Comment by unarist** [comment]
"Reject media" destroys all existing media, but removing it doesn't cause re-fetching. Also we don't update profiles in ActivityPub as often as OStatus. 

So we may want to enqueue new worker which does `rake mastodon:media:redownload_avatars DOMAIN=example.com`. (although it doesn't seem to check media blocks...)

**Comment by nolanlawson** [comment]
Hmmm, can I fix this manually by resubscribing to the instance in the admin interface?

**Comment by nolanlawson** [comment]
BTW you're right; some media blocks don't seem to be getting fixed. (I.e. the media is not embedded; it just shows links to the original media URL.) It appears that some older posts still have the media blocked, although new posts are unaffected.

**Comment by nolanlawson** [comment]
Avatars are fixed manually by doing `rake mastodon:media:redownload_avatars DOMAIN=example.com`, thanks for the tip! :+1:

**Comment by vmstan** [comment]
This issue was filed against a now unsupported version of Mastodon. In an effort to clean up stale bugs, if this can be reliably reproduced on an updated version please open a new issue with updated logs/troubleshooting and we will take a look. Thank you.
